addons/models/0001-atlas-authz/0_Base.json

```diff
@@ -0,0 +1,109 @@
+{
```

Comment:
Lets not couple typedefs with service code. And keep typedef lifecycle out, we should decouple them as much as possible. Which will help us break atlas in microservices in future.

Please take a call, depending on strict need basis

Reply:
@n5nk , I feel a strict need to keep these typeDefs here
Justification: We are bootstrapping default access policies at Atlas startup which needs typeDefs already created. AuthPolicy & AuthService are both we want to inherit from Asset, so had to move Asset & referenceable here.
But we have copied them with minimal required attributeDefs, other attributeDefs will be created when typeDef seeder will be ran